classes/Animal.py

```diff
@@ -0,0 +1,5 @@
+class Animal:
```

Comment:
have you considered a namedtuple or data class, considering that animals (so far) don't actually have methods? is it intentional?